OsmAnd/res/values/strings.xml

```diff
@@ -11,26 +11,26 @@
 	Thx - Hardy
 -->
 
-    <string name="shared_storage_warning_description">Starting November 2021 OsmAnd 4.2 will lose access to Shared storage, due to changed storage access rules in Android.\n\nPlease change the storage from Shared storage to available storage, if this is not done the app will lose access to your data: including offline maps, GPX tracks and everything else.</string>
-    <string name="search_history_is_disabled_descr">You can enable search history in Settings</string>
-    <string name="markers_history_is_disabled_descr">Enable history to get list of visited map markers.</string>
-    <string name="history_is_disabled_descr">Enable history to get quicker access to last used results.</string>
-    <string name="is_disabled">%1$s is disabled</string>
-    <string name="clear_all_history_warning">Are you sure you want to clear all history in the app?\n\nThe search, navigation and marker history will be cleared.</string>
+    <string name="shared_storage_warning_description">If you are using Android 4.2, change from shared to available storage before Nov. 2021 to retain access to offline maps, GPX tracks, etc.</string>
+    <string name="search_history_is_disabled_descr">You can turn on search history in the settings</string>
+    <string name="markers_history_is_disabled_descr">Turn on history to get a list of visited map markers.</string>
+    <string name="history_is_disabled_descr">Turn on history to access last used results quicker.</string>
+    <string name="is_disabled">%1$s is off</string>
+    <string name="clear_all_history_warning">Clear all app history including search, navigation and marker history?</string>
     <string name="clear_all_history">Clear all history</string>
     <string name="history_actions_descr">You can export history into a file.</string>
     <string name="map_markers_history">Map markers history</string>
     <string name="navigation_history">Navigation history</string>
-    <string name="history_preferences_descr">You can disable history logging individually for each history type.</string>
+    <string name="history_preferences_descr">You can turn off logging for each history type.</string>
     <string name="shortcut_start_recording">Start Recording</string>
     <string name="shortcut_navigate_to">Navigate To…</string>
-    <string name="shortcut_navigate_to_work">Navigate to Work</string>
-    <string name="shortcut_navigate_to_home">Navigate to Home</string>
+    <string name="shortcut_navigate_to_work">Work</string>
```

Comment:
Not 100% sure about these two.
Is "Navigate to…" visible?

Reply:
? this is correct they are all visible and Work is very confusing name

Reply:
Not a big loss, so reverting these for now.